### DISCUSSION TITLE
Lively hardware api values

### DISCUSSION THREAD

**Discussion by noobavenger** [rocksdanister/lively#2365] [General]
What exactly are the values returned by livelySystemInformation(data)? Is obj.CurrentCpu a percentage value or some scaled version of cpu speed? And how often is liveySystemInformation called?

**Comment by rocksdanister** [comment]
https://github.com/rocksdanister/lively/wiki/Web-Guide-V-:-System-Data#--system-information
https://github.com/rocksdanister/lively/blob/core-separation/src/Lively/Lively.Common.Services/Hardware/CounterHardwareUsageService.cs
